awscli/examples/configure/get/_examples.rst

```diff
@@ -24,12 +24,12 @@ The following commands would have the corresponding output::
     testing_access_key
 
     $ aws configure get profile.testing.aws_access_key_id
-    default_access_key
+    testing_access_key
 
     $ aws configure get preview.cloudsearch
     true
 
     $ aws configure get preview.does-not-exist
-    $
```

Comment:
I made the change as I saw there is always an empty line between prompts. I ran my test with PS1 set to `"\n$ "`. I'm also okay to keep this and amend my commit.